tRecorderApi/api/views/__init__.py

```diff
@@ -18,3 +18,5 @@
 from version_list import *
 from update_project import *
 from version_list import *
```

Comment:
@lcheng49 can you just remove this unused import for me. It's in there twice - we only need one